django/applications/catmaid/control/cropping.py

```diff
@@ -496,14 +480,16 @@ def extract_substack_no_rotation(job) -> List:
                                      (estimated_total_size,
                                       settings.GENERATED_FILES_MAXIMUM_SIZE))
                 # Draw the image onto result image
-                cropped_slice.composite( image, ip.x_dst, ip.y_dst, co.OverCompositeOp )
+                cropped_slice.paste(image, ip.x_dst, ip.y_dst)
```

Comment:
This doesn't seem to work like that and I believe you need to do this:
```
cropped_slice.paste(image, (ip.x_dst, ip.y_dst))
```
At least that worked for me.

Reply:
Since I merged this branch already, I fixed this in `dev`.